### DISCUSSION TITLE
Can't find a shortcut key for deepin global search that is introduced in 20.2.4

### DISCUSSION THREAD

**Discussion by zbandhan** [linuxdeepin/developer-center#2868] [Q&A | 问答板块] [answered]
Thanks to Deeping team for giving us such a nice looking OS. I noticed a global search comes with Deepin in the new release that is really useful and I need a shortcut key to quick navigate into it but I didn't find any. 

Is there any solution for it or is it something you guys are still working on?

Regards.

**Comment by BLumia** [comment, marked as answer]
Hi, sorry for the super late reply.

By default the <kbd>Shift+Space</kbd> shortcut is for the global search (so called "Grand Search"). If that shortcut is not available in your distro by default, you can also add a custom shortcut that executes `dde-grand-search` for that.
